src/search/_Search.jsx

```diff
@@ -76,12 +77,12 @@ const highlight = (key, result) => {
   return result.item[key];
 };
 
-const searchItem = result => (
+const searchItem = (result, theme) => (
```

Comment:
you can clean all this up by .... puttting the theme.className at the top of the component :)